main/src/main/js/launch/src/components/NextSteps/NextSteps.js

```diff
@@ -42,11 +42,13 @@ const NextSteps = ({
     }, [info.type, cloneUrl, name])
 
     const cdCommand = useMemo(() => {
-        const win = `cd ${name}`
-        const nix = `./cd ${name}`
+        const cd = `cd ${name}`
```

Comment:
I think we has just put the relative path on the wrong part. It should have been.
```
        const win = `cd ${name}`
        const nix = `cd ./${name}`
```

Reply:
@eahrold  Not sure if I understand. Do you think, that in this PR, this line: 

```
        const cd = `cd ${name}`
```

should be replaced with: 

```
        const cd = `cd ./${name}`
```

Reply:
@sdelamo : although `cd ./${name}` would work it is rather uncommon. The prefix `./` is usually only used when referencing executables because it is needed when `.` (current directory) is not in the PATH (which is often the default).

IMHO, I'd still merge the pull request as it is.

Reply:
I’m fine with the pr as is. Was more just making a comment as to what got mixed up.  And while not necessary to have the ‘./‘ for cd it just lent a touch of context that the command was relative to the download path, not just your current path, and probably a case of overthinking.